admin.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
```

Comment:
there should be no space before the head tag. Please remove the space before the opening and the closing head tag.